formalize_convert/main.py

```diff
@@ -176,13 +192,19 @@ def convert(user_query: str, formalized_query: str):
 
             s = query_line[i].split('=')[1]
 
-            if bool(re.match(only_variables_pattern, s)):
-                return False
-            else:
-                if query_line[i] in user_query:
-                    trs += query_line[i] + '\n'
+            if query_line[i].replace('*', '').replace('{', '').replace('}', '') in user_query:
```

Comment:
nitpick: `re.sub(r'[\*{}]', '', query_line[i]) in user_query `

Reply:
@pank-art

Reply:
```suggestion
            if re.sub(r'[\*{}]', '', query_line[i]) in user_query:
```